src/Plugin/Commerce/ShippingMethod/CanadaPost.php

```diff
@@ -102,6 +111,14 @@ public static function create(ContainerInterface $container, array $configuratio
    */
   public function defaultConfiguration() {
     return [
+      'api' => [
+        'customer_number' => '',
+        'username' => '',
+        'password' => '',
+        'contract_id' => '',
+        'mode' => 'test',
+        'log' => 'test',
```

Comment:
**CHANGE REQUEST**

Not sure if the default value for `log` makes sense, is it not an array like `['request', 'response']`? Should be set to an empty array?